### PR TITLE
Add bloggpost `Creating a postman collection from swagger`

### PR DESCRIPTION
**Pull request by gadapunkfun**
## PR Description
This PR will add the blog post `creating a postman collection from swagger`, included are some pictures.

### CODE REVIEW

**Comment by Dunklas** [comment]
I think you'll need to add the images to the `static` folder.
Other than that it looks good to me! Awesome! 💯

**Review by Dunklas — DISMISSED** [review]

File: content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+This is probably not news to anyone that has been working as developer consultant.
+Large existing code base; in our case it's an application that has existed for about 2,5 years and during that time it has developed 
+several services and therefore has many endpoints.
+
+Therefore when me and my colleagues had to start developing new features it was hard to know what was available and what wasn't with over 50 different endpoints to hit.
+
+# Solution
+
+We know and love postman so we wanted to find a way to use that to share an equal standing ground consisting of shared knowledge.
```
Comment (omersimsek29) [inline content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md:22]: postman -> Postman

File: content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+After frantically pressing buttons and looking around the postman application we found the convinent button labeled `Import` and that's when me and my colleague said aha! 💡 </br>
```
Comment (omersimsek29) [inline content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md:23]: convinent -> convenient

File: content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+After opening that menu option we saw that importing data from swagger was possible and the project we were working with had that one requirement.
```
Comment (omersimsek29) [inline content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md:24]: swagger -> Swagger

File: content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+> ![import window](/blog/swagger-to-postman/importing.png)
+
+> ![link added to postman import window](/blog/swagger-to-postman/link.png)
+4. Voilá!
+> ![postman with new collection](/blog/swagger-to-postman/imported.png)
+
+# Final Notes
+
+This is a fantastic feature to keep in mind not only for scenarios similar to this it's also good if you want to use other tools that postman offers.
+As an example you can create simple JavaScript scripts to set authentication token for your requests, something you clearly can't do in swagger, making local development environment that much easier and looking more proffesional. (thanks @BunnyFiscuit for showing me that trick).
```
Comment (omersimsek29) [inline content/posts/2023-05-04_creating-a-postman-collection-from-swagger.md:44]: proffesional -> professional

**Review by omersimsek29 — CHANGES_REQUESTED** [review]
Kolla mina kommentarer😊 

Jag rekommenderar Grammarly eller till och med ChatGPT för att finslipa posten. Annars tycker jag att texten är nice.

**Review by Dunklas — APPROVED** [review]

**Pull request merged**